tutorial/write-custom-widget.md

```diff
@@ -107,3 +107,178 @@ Take care not to keep any important state in your renderer - animation tickers
 are well suited to that location but user state would not be. A widget that is
 hidden may have it's renderer destroyed and if it is shown again the new renderer
 must be able to reflect the same widget state.
+
+### Example Widget
+
+The following covers in detail the creation of a simple Widget and it's Renderer.
+
+#### Create a Widget
+
+So the widget requires the`widget.BaseWidget`. For example:
+
+```go
+type MyWidget struct {
+	widget.BaseWidget           // Inherit from BaseWidget
+	minSize           fyne.Size // The minimum size of the widget
```

Comment:
Storing this internally could be a distraction from the example - it is not a normal thing to do